02_cancer_type_classification/run_cancer_type_classification.py

```diff
@@ -42,7 +42,7 @@ def process_args():
     p.add_argument('--debug', action='store_true',
                    help='use subset of data for fast debugging')
     p.add_argument('--feature_selection',
-                   choices=['mad', 'pancan_f_test', 'median_f_test', 'mad_f_test'],
+                   choices=['mad', 'pancan_f_test', 'median_f_test', 'random'],
```

Comment:
I get that you're adding random feature selection, but can you explain why you're getting rid of the mad_f_test?

Reply:
`mad_f_test` didn't really make any sense to begin with - the idea was to select features that have low variability across cancer types, but what that ends up doing is just picking features that aren't correlated with the label in any cancer type, since if a feature has ~0 correlation in all cancer types it will have low variance.

I had been including `mad_f_test` as a baseline/lower bound, but just selecting random features pretty much always outperforms it, and that seems like a more sensible baseline to use going forward.